src/onnxruntime.cc

```diff
@@ -682,10 +682,40 @@ ModelState::AutoCompleteConfig()
   OrtAllocator* default_allocator;
   std::string model_path;
   {
+    TRITONSERVER_InstanceGroupKind kind;
```

Comment:
Initialize this to CPU by default and change to GPU when 
```
      if (kind_str == "KIND_GPU" || kind_str == "KIND_AUTO") {
        kind = TRITONSERVER_INSTANCEGROUPKIND_GPU;
        break;
      }
```

Reply:
Updated to `kind = TRITONSERVER_INSTANCEGROUPKIND_CPU` for initialization

Reply:
Sorry I read this comment wrong the first time. You and [rmccorm4](https://github.com/rmccorm4) had the same suggestion.